### DISCUSSION TITLE
How are the bounds for the regularization parameter lambda computed?

### DISCUSSION THREAD

**Discussion by Timvanderschaaf** [facebookexperimental/Robyn#779] [General]
I cannot figure out what the reasoning is behind the bounds for the regularization parameter lambda in the Ridge Regression.

Please see below snippets of code from _model.R_. The first code snippet (see also last snippet) uses a function to compute a list of lambdas, however...
![image](https://github.com/facebookexperimental/Robyn/assets/91058747/30c080a2-11f4-475e-a12b-105b58b55831)

... in the _model_refit_ function, uses _lambda_hp_ and I cannot find out where this value comes from?
![image](https://github.com/facebookexperimental/Robyn/assets/91058747/b16083b8-1149-4b20-8aa6-e20fc7145097)
![image](https://github.com/facebookexperimental/Robyn/assets/91058747/8135349b-3408-4c37-8260-8cce08386f08)

I would like to understand how the list of possible lambda values exactly is being constructed.

**Comment by gufengzhou** [comment]
Sorry for the late reply. `lambda_seq` is derived directly from the glmnet package. Not sure about your question tho.
